tests/rptest/tests/read_replica_e2e_test.py

```diff
@@ -103,14 +104,16 @@ def create_read_replica_topic_success():
             else:
                 return True
 
+        assert self.redpanda and self.redpanda.s3_client
         # wait until the read replica topic creation succeeds
         wait_until(
             create_read_replica_topic_success,
             timeout_sec=
             60,  #should be uploaded since cloud_storage_segment_max_upload_interval_sec=5
             backoff_sec=5,
-            err_msg=
-            f"Could not create read replica topic. Most likely because topic manifest is not in S3, in S3 bucket: {list(self.redpanda._s3client.list_objects(self.s3_bucket_name))}"
+            err_msg="Could not create read replica topic. Most likely " +
```

Comment:
nit: I think we can get away without adding (`+`) strings explicitly. Since we are in parens, strings adjacent to each other on separate lines will be concatenated by the interpreter. But being explicit is fine too.

Reply:
Oh.. good tip.